spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/support/RateLimitEvent.java

```diff
@@ -0,0 +1,42 @@
+package com.marcosbarbero.cloud.autoconfigure.zuul.ratelimit.support;
+
+import com.marcosbarbero.cloud.autoconfigure.zuul.ratelimit.config.properties.RateLimitProperties.Policy;
+import com.marcosbarbero.cloud.autoconfigure.zuul.ratelimit.filters.RateLimitPreFilter;
+import java.util.Objects;
+import org.springframework.context.ApplicationEvent;
+
+/**
+ * Event raised when a rate limit exceeded.
+ *
+ * @author vasilaio
+ */
+public class RateLimitEvent extends ApplicationEvent {
```

Comment:
Is it possible to make this class final? I want to prevent people from overriding it.